src/server/pfs/server/api_server.go

```diff
@@ -184,19 +184,7 @@ func (a *apiServer) FinishCommit(ctx context.Context, request *pfs.FinishCommitR
 	defer func(start time.Time) { a.Log(request, response, retErr, time.Since(start)) }(time.Now())
 	if err := a.txnEnv.WithTransaction(ctx, func(txn txnenv.Transaction) error {
 		return txn.FinishCommit(request)
-	}, func(txnCtx *txncontext.TransactionContext) (string, error) {
-		// FinishCommit in a transaction by itself has special handling with regards
```

Comment:
Is this comment just totally unnecessary now?

Reply:
The special case still applies, but it occurs in the background process that finishes the commit rather than here. There is a note about it where the commit is finished.